app/controllers/candidate_interface/course_choices/course_selection_controller.rb

```diff
@@ -77,6 +135,22 @@ def redirect_to_review_page_if_course_already_added(application_form, course_id)
           redirect_to candidate_interface_course_choices_review_path
         end
       end
+
+      def return_to_after_edit(default:)
+        if redirect_back_to_application_review_page?
+          { back_path: candidate_interface_application_review_path, params: redirect_back_to_application_review_page_params }
+        else
+          { back_path: default, params: {} }
+        end
+      end
+
+      def redirect_back_to_application_review_page_params
+        { 'return-to' => 'application-review' }
+      end
+
+      def redirect_back_to_application_review_page?
+        params['return-to'] == 'application-review' || params[:return_to] == 'application-review'
```

Comment:
Unfortunately, I don't think we can because `'return-to'` and `:return_to` aren't analagous.

Out of interest I tried adding params[:return_to] == 'application-review'  to the method on the `CandidateInterfaceController` instead and it appears to not affect the functionality of any of the other back links.

Shall we take a chance or merge in as is? cc @davidgisbey